crates/uv-cli/src/lib.rs

```diff
@@ -567,6 +567,19 @@ pub enum ProjectCommand {
     )]
     Remove(RemoveArgs),
     /// Update the project's environment (experimental).
+    ///
+    /// Syncing ensures that all dependencies of the project are installed and
+    /// up to date with the lockfile. Syncing also removes packages that are not
+    /// declared as dependencies of the project.
+    ///
+    /// If the project virtual environment (`.venv`) does not exist, it will be
+    /// created.
+    ///
+    /// The project is re-locked before syncing unless the `--locked` or
+    /// `--frozen` flag is provided.
```

Comment:
"flags are"?

Reply:
Only one can be provided, idk e.g. it shortens to `unless the `--locked` flag is provided`. Could go either way, don't know what is "correct".